### PR TITLE
Improve compatibility for MW1.35+

### PR DESCRIPTION
**Pull request by StarHeartHunt**
Fix 'User Comments' link in the sidebar & 'Comment Control' action

### CODE REVIEW

File: includes/Hooks.php [hunk 1]
```diff
@@ -87,7 +87,49 @@ public static function onBaseTemplateToolbox(\BaseTemplate &$baseTemplate, array
 			$toolbox['usercomments']['id'] = 't-usercomments';
 		}
 	}
+	
+	public static function onSidebarBeforeOutput(\Skin $skin, &$sidebar) {
+		if ( version_compare( MW_VERSION, '1.35', '<' ) ) {
+			return false;
+		}
```
Comment (nbdd0121) [inline includes/Hooks.php:94]: This comparison seems to be unnecessary.

File: includes/Hooks.php [hunk 2]
```diff
@@ -87,7 +87,49 @@ public static function onBaseTemplateToolbox(\BaseTemplate &$baseTemplate, array
...
+				'href' => \SpecialPage::getTitleFor('FlowThreadControl', $title->getPrefixedDBKey())->getLocalURL()
+			];
+		}
+
+		return true;
+	}
+	
+	//Deprecated in MW 1.35+. See https://www.mediawiki.org/wiki/Manual:Hooks/SkinTemplateOutputPageBeforeExec
+	//Use onSkinTemplateNavigation_Universal instead.
+
```
Comment (nbdd0121) [inline includes/Hooks.php:132]: Is there a reason not to remove onSkinTemplateOutputPageBeforeExec all together?

**Review by nbdd0121 — CHANGES_REQUESTED** [review]
Thanks for the PR. Could you address the two comments please?

**Review by StarHeartHunt — COMMENTED** [review]
That's my fault.
I have removed onSkinTemplateOutputPageBeforeExec from the code in the latest commit.

**Review by StarHeartHunt — COMMENTED** [review]
Removed redundant code.

**Comment by nbdd0121** [comment]
Squashed and merged

**Pull request closed without merging**